README.md

```diff
@@ -0,0 +1,15 @@
+# Low Power library
```

Comment:
```suggestion
# Arduino Low Power library
```
The official library name is defined at https://github.com/arduino-libraries/ArduinoLowPower/blob/c1b24fb45651b5eef859ae6fb0da76c19560f2ff/library.properties#L1
This is the name to search for in the Arduino Library Manager index whether using the Arduino IDE, Arduino Pro IDE, or arduino-cli (`arduino-cli lib install "Arduino Low Power"`)